lib/GrepCpan/std.pm

```diff
@@ -0,0 +1,20 @@
+package GrepCpan::std;
+
+use strict;
+use warnings;
+
+sub import {
+
+    # auto import strict and warnings to our caller
+
+    warnings->import();
+    strict->import();
+
+    require feature;
+    feature->import( ':5.30', 'signatures' );
```

Comment:
I guess this could be 5.36, but maybe not if you want it to be runnable outside of Docker.

Reply:
yeap both could work